CHANGELOG.md

```diff
@@ -1126,12 +1126,6 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
   - Add/move/remove from group: removed completely (functionality still available through group interface)
 - We removed the option to change the column widths in the preferences dialog. [#4546](https://github.com/JabRef/jabref/issues/4546)
 
-## Older versions
```

Comment:
I would keep it

Reply:
You mean "at that place" or "generally"?

Here the rendered version of the updated CHANGELOG.md:

![image](https://github.com/JabRef/jabref/assets/1366654/0cf59d5c-8072-4f6f-9e86-5c7b7eaa01d3)

Reply:
The heading `Older Versions` can go +1, as it breaks the hierarchical tree. Where would be otherwise the subdistinctions to `Unreleased`?

Reply:
Why is this a problem having the old versions still there?

Reply:
I talking about the heading "older versions", not about the headings of the older versions itself.

Reply:
Yes I understood, but why remove this? It's perfectly fine.